demo/admin/src/products/ProductsGrid.tsx

```diff
@@ -90,6 +90,22 @@ export function ProductsGrid() {
         { field: "description", headerName: "Description", width: 150 },
         { field: "price", headerName: "Price", width: 150, type: "number" },
         { field: "type", headerName: "Type", width: 150, type: "singleSelect", valueOptions: ["Cap", "Shirt", "Tie"] },
+        {
+            field: "additionalTypes",
+            headerName: "Additional Types",
+            width: 150,
+            renderCell: (params) => <>{params.row.additionalTypes.join(", ")}</>,
+            filterOperators: [
+                {
+                    value: "contains",
+                    getApplyFilterFn: (filterItem) => {
+                        throw new Error("not implemented, we filter server side");
+                    },
+                    InputComponent: GridFilterInputSingleSelect,
+                },
+            ],
```

Comment:
Should this be best practice?

Reply:
I think there are two options:

1. try to add a new `type` to DataGrid (called eg. simpleSelectMultiple)
2. always use code similar to this one

for this PR (which aims at the api side) :two: should be good enough. When we want to support this in admin crud generator, this can be improved.

Reply:
As far as I understand it, this function will only be called for client-side filtering, right? Then it shouldn't be a problem IMO

Reply:
> Then it shouldn't be a problem

yes. That doesn't make it best practice though :)